invenio_app_ils/circulation/loaders/schemas/json/common.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio App ILS circulation common loader JSON schema."""
+
+from invenio_records_rest.schemas import RecordMetadataSchemaJSONV1
+from marshmallow import fields
+
+
+class LoanCommonSchemaV1(RecordMetadataSchemaJSONV1):
+    """Loan common schema."""
+
+    class Meta:
+        """Meta attributes for the schema."""
+
+        from marshmallow import EXCLUDE
+        unknown = EXCLUDE
+
+    def get_pid_field(self):
```

Comment:
This is not needed anymore https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/config.py#L322